cvmfs/sync_mediator.cc

```diff
@@ -75,6 +75,12 @@ void SyncMediator::Add(const SyncItem &entry) {
     return;  // Ignore markers.
   }
 
+  if (entry.IsSpecialFile() && !entry.IsWhiteout()) {
```

Comment:
Perhaps add a comment here that whiteout entries on OverlayFS are character device files